ansible/roles/ipsilon/tasks/main.yml

```diff
@@ -110,6 +111,31 @@
     - httpd_can_network_connect
     - httpd_use_fusefs
 
+- name: Remove default static alias from httpd conf
+  replace:
+    path: /etc/ipsilon/idp/idp.conf
+    regexp: 'Alias \/idp\/ui \/usr\/share\/ipsilon\/ui'
+    replace: '#Alias /idp/ui /usr/share/ipsilon/ui'
+
+- name: Add static alias for Fedora theme in httpd conf
+  lineinfile:
+    path: /etc/ipsilon/idp/idp.conf
+    insertbefore: BOF
+    line: Alias /idp/ui /usr/share/ipsilon/themes/Fedora
+
+- name: Add static alias for Fedora theme in httpd conf
+  lineinfile:
+    path: /etc/ipsilon/idp/idp.conf
+    insertbefore: BOF
+    line: Alias /ui /usr/share/ipsilon/themes/Fedora
```

Comment:
What's the reason for having both aliases `/idp/ui` and `/ui` point to the same directory?

Reply:
there is an issue in the fedora theme (layout.html) where the top logo is referenced like this:

    <img alt="logo" src="{{ basepath }}/ui/res/fedora-authn-logo-white.png?v=dcabcce440f62c6282cfffb82f2f5614" height=40px />

and the custom css is refernced like this:

    <link href='/ui/res/login.css' rel='stylesheet' type='text/css'>

im planning on doing work on the fedora ipsilon theme next and will add the {{ basepath }} to the css. but just hacked the extra alias in there to get things working for now (will remove it later.)

Reply:
Ah gotcha.